dat/outfits/unique/fuel_hull_plating.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<outfit name="Fuel Hull Plating">
```

Comment:
Probably needs a better name too. Maybe "Fuelshell Plating" and maybe give it a majer name and some more in-depth description.